var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -76,6 +76,8 @@ def install(self, spec, prefix):
 
         if '+libiberty' in spec:
             configure_args.append('--enable-install-libiberty')
+            configure_args.append('--disable-nls')
+ 
```

Comment:
Just delete the blank line you added. According to PEP-8, you should only have a single blank line inside of a class. Outside of a class you should have exactly 2. Flake8 checks for PEP-8 compliance.